src/main/java/org/broadinstitute/hellbender/tools/exome/SegmentUtils.java

```diff
@@ -302,7 +351,32 @@ public static void writeACNVModeledSegmentFile(final File outFile,
                                 .append(String.format("%6.4f", segment.getMinorAlleleFractionPosteriorSummary().lower())
                                         .replaceAll("\\s+", ""))
                                 .append(String.format("%6.4f", segment.getMinorAlleleFractionPosteriorSummary().upper())
-                                        .replaceAll("\\s+","")));
+                                        .replaceAll("\\s+", "")));
+    }
+
+    /**
+     * Read ACNV file into ACNV modeled segments.
+     *
+     * @param acnvSegFile File in the ACNV format (*-final.seg).  Must be readable file and never {@code null}
+     * @return Never {@code null}.  Can be empty list if no entries exist in the input file.
+     */
+    public static List<ACNVModeledSegment> readACNVModeledSegmentFile(final File acnvSegFile) {
```

Comment:
Should also be a one-liner:
`return readSegmentFile(acnvSegFile, SegmentTableColumns.ACNV_MODELED_SEGMENT_COLUMN_NAME_ARRAY, SegmentUtils::toACNVModeledSegment);`

Also it seems cleaner to me to have all of the parameters for these readXFile methods be named "segmentsFile" or something standard.